toucan_connectors/common.py

```diff
@@ -496,6 +500,8 @@ def pandas_read_sql(
         query = query.replace("%%", "%")
         query = re.sub(r"%[^(%]", r"%\g<0>", query)
         df = pd.read_sql(query, con=con, params=params, **kwargs)
+        if "?column?" in df.columns:
+            raise SelectedColumnInterpolationError("Selected column names cannot be parametrized.")
```

Comment:
Seems dangerous to me: the query has already been executed. Also, the column name might change